### DISCUSSION TITLE
Eachine E130 protocol

### DISCUSSION THREAD

**Discussion by pavelajakdal** [pascallanger/DIY-Multiprotocol-TX-Module#823] [Q&A]
Hello,
please, I have the same problem as other users, sometime between 2022 and 2023. I can't pair the Eachine E130 helicopter with the Eachine (Radiomaster) TX16S. I have successfully installed the latest firmware and scripts. Futaba protocol mentioned in the manual is broken, the E130 protocol according to Youtube (video tutorial about one year old) is broken. The original transmitter is functional. The supplier of both Banggood products refused help or contact to the manufacturer. The manufacturer of the Eachine E130 helicopter must have secretly changed something. It would be nice if someone clever would create an OpenTX protocol that would work for this variant as well. The throttle right control on the original transmitter is unusable nonsense - have you seen anyone control the cyclic with both hands separately? I otherwise use throttle right. Thank you for your help, kind regards.

**Comment by pascallanger** [comment]
You are the first one to report that the Eachine E130 is using a different protocol at least here. Unless I get access to one of these new heli version there is nothing I can do. If you are willing to ship the TX + model to France or sponsor the purchase of the heli I can have a look.

**Comment by pavelajakdal** [comment]
Dear Mr. Langer,

thank you for your willingness to help. I am a simple modeler and I bought 
the helicopter for fun. I have found similar problems to mine here:

https://www.rcgroups.com/forums/showthread.php?3623759-Radiomaster-TX16S-
Owner-s-Thread/page368#post50295143





I have the following idea. I have exchanged several messages with Banggood, 
where I sometimes buy models and where I bought the Eachine E130 helicopter.
It's the world's leading store in this stuff and proudly proclaims it. They 
sell both the OpenTX transmitters and the Eachine E130 helicopters all over 
the world. Last time they wrote to me they could not help me regarding the 
problem mentioned. What they could do, however, in my opinion, is to send 
you this Eachine E130 helicopter set free of charge to review. Especially 
when the instructions spread nonsense about Futaba protocol compatibility 
and a bind button in the helicopter, which is not true at all. If they are 
selling the helicopter, they could do something to make the goods more true 
to reality, that is, make it truly compatible.




If you allow me, I would write to them about you and your offer and ask for 
their help. Do you agree? I am still sending pictures of my transmitter - 
the protocols that did not work for this model.




I wish you success, thank you.

Sincerely

Pavel Václavík

CZ






---------- Původní e-mail ----------
Od: pascallanger ***@***.***>
Komu: pascallanger/DIY-Multiprotocol-TX-Module <DIY-Multiprotocol-TX-Module@
noreply.github.com>
Kopie: pavelajakdal ***@***.***>, Author ***@***.***
com>
Datum: 22. 2. 2023 18:41:12
Předmět: Re: [pascallanger/DIY-Multiprotocol-TX-Module] Eachine E130 
protocol (Discussion #823)
"


You are the first one to report that the Eachine E130 is using a different 
protocol at least here. Unless I get access to one of these new heli version
there is nothing I can do. If you are willing to ship the TX + model to 
France or sponsor the purchase of the heli I can have a look.

—
Reply to this email directly, view it on GitHub
(https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/discussions/823#discussioncomment-5079930)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/A6BCAGVGP2DHIKWIBZEZIJLWYZFTJANCNFSM6AAAAAAVC3FZIU)
.
You are receiving this because you authored the thread. Message ID: <
pascallanger/DIY-Multiprotocol-TX-Module/repo-discussions/823/comments/
***@***.***>
"

**Reply by pascallanger** [reply]
You can always try. They've never accepted to send me any models... Maybe if customers are asking it will be different, so try and let me know.

**Reply by pascallanger** [reply]
Any luck?

**Comment by pavelajakdal** [comment]
Hello,
I'm sorry for not writing to you. I'm waiting for a statement from Bangood, 
they say they put it in the relevant department. Since they don't answer, I 
will write to them that they can't leave me like this without an answer to 
Mr. Langer. Maybe they can tell each other. I'll get back to you anyway.

Thank you for your help, have a good day.

With regards

Pavel Václavík 


---------- Původní e-mail ----------
Od: pascallanger ***@***.***>
Komu: pascallanger/DIY-Multiprotocol-TX-Module <DIY-Multiprotocol-TX-Module@
noreply.github.com>
Kopie: pavelajakdal ***@***.***>, Author ***@***.***
com>
Datum: 14. 3. 2023 12:11:09
Předmět: Re: [pascallanger/DIY-Multiprotocol-TX-Module] Eachine E130 
protocol (Discussion #823)
"


Any luck?

—
Reply to this email directly, view it on GitHub
(https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/discussions/823#discussioncomment-5301997)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/A6BCAGWMHZ52XY6B77FN7WTW4BG4TANCNFSM6AAAAAAVC3FZIU)
.
You are receiving this because you authored the thread. Message ID: <
pascallanger/DIY-Multiprotocol-TX-Module/repo-discussions/823/comments/
***@***.***>
"

**Comment by pavelajakdal** [comment]
I sent this to Banggood:


Hello, it's been a long time since your last message. I still don't have the
answer from you that I asked for and that I need to give to the creator of 
the OpenTX protocols, Mr. Langer. I trust that you have the success of your 
customers in mind and that you are therefore not indifferent to the fact 
that the RC protocol in the helicopter does not match the RC protocol in its
manual. I await your response to provide Mr. Langer with the requested study
material.

Thank you, kind regards

PV


---------- Původní e-mail ----------
Od: pascallanger ***@***.***>
Komu: pascallanger/DIY-Multiprotocol-TX-Module <DIY-Multiprotocol-TX-Module@
noreply.github.com>
Kopie: pavelajakdal ***@***.***>, Author ***@***.***
com>
Datum: 14. 3. 2023 12:11:09
Předmět: Re: [pascallanger/DIY-Multiprotocol-TX-Module] Eachine E130 
protocol (Discussion #823)
"


Any luck?

—
Reply to this email directly, view it on GitHub
(https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/discussions/823#discussioncomment-5301997)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/A6BCAGWMHZ52XY6B77FN7WTW4BG4TANCNFSM6AAAAAAVC3FZIU)
.
You are receiving this because you authored the thread. Message ID: <
pascallanger/DIY-Multiprotocol-TX-Module/repo-discussions/823/comments/
***@***.***>
"

**Comment by pavelajakdal** [comment]
Hello, Mr. Langer,



I have tried several times to get Banggood to send you a study model to add 
the RC protocol to OpenTX. This company, to my knowledge, misleads customers
by listing a different protocol in the manual than the one that is actually 
in the model and the interfacing with the OpenTX transmitter (which they 
also sell) has failed. You can read the style of Banggood's response to the 
customer yourself in the copy - see below. They have replied to me more than
once anyway. They seem to just want to sell at any price, not solve the 
problem they cause the customer. It strikes me as their mockery of the 
customer, but judge for yourself.

Maybe you can get the model yourself.

Sincerely

Pavel Václavík





---------------------


  Načíst externí obrázkyZ důvodu bezpečnosti jsme nenačetli externí obrázky 
  obsažené v tomto e-mailu.

        
        Dear Banggood,
        
        in your last reply to me about the malfunction of pairing your 
        Eachine E130 product with your Eachine (Radiomaster) TX16S you wrote
        that you cannot help me. I have made efforts in this matter and I 
        have happy news for you: I have contacted the well-known creator of 
        the OpenTX system that both your mentioned products should use. Mr. 
        Pascal Langer, has promised me that he will make an effort to 
        investigate and, if possible, add the new Eachine E130 helicopter 
        protocol to the OpenTX protocol system. As you are fully aware, 
        contrary to its manual, the helicopter does not respond to the 
        Futaba protocol, nor to the E129 protocol, because apparently its 
        manufacturer has changed the protocol. This has disappointed more of
        its buyers. Now is the chance to fix it. On your part, this would 
        involve sending one study piece of the aforementioned E130 
        helicopter, including the set transmitter, to Mr. Langer in France 
        free of charge so that he can take the necessary steps to create an 
        RC protocol. I trust that you will accept this effort positively 
        and, as a world supplier of these products, will help to remedy this
        situation.
        
        Please let me know your opinion, I trust it will be productive.
        
        Sincerely
        
        Pavel Vaclavik
        






***@***.***

 
 
 
 Re: 未下载邮件:Re: 未下载邮件:Eachine E130 binding
 


  Načíst externí obrázkyZ důvodu bezpečnosti jsme nenačetli externí obrázky 
  obsažené v tomto e-mailu.

        Hello, Banggood,
        you are still silent, but I would like to know from you when and how
        you will answer me? What should I tell Mr. Langer from OpenTx? Are 
        you going to leave your most valued customer without an answer to 
        the important matter of misleading him into buying your goods (RC 
        protocol)? That would be most unfortunate.
        
        Sincerely.
        
        Pavel Vaclavik
        
        customer
        













 
 Re: 未下载邮件:Re: 未下载邮件:Eachine E130 binding
 

 
 
 
 


  Dear Customer,
  

        
        
        
        
        Thank you for your e-mail.
        
        
        
        
        If you have any other inquiries, please do not hesitate to contact 
        us at any time.
        
        
        
        
        Best regards,
        
        Banggood Team
        







---------- Původní e-mail ----------
Od: pascallanger ***@***.***>
Komu: pascallanger/DIY-Multiprotocol-TX-Module <DIY-Multiprotocol-TX-Module@
noreply.github.com>
Kopie: pavelajakdal ***@***.***>, Author ***@***.***
com>
Datum: 14. 3. 2023 12:11:09
Předmět: Re: [pascallanger/DIY-Multiprotocol-TX-Module] Eachine E130 
protocol (Discussion #823)
"


Any luck?

—
Reply to this email directly, view it on GitHub
(https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/discussions/823#discussioncomment-5301997)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/A6BCAGWMHZ52XY6B77FN7WTW4BG4TANCNFSM6AAAAAAVC3FZIU)
.
You are receiving this because you authored the thread. Message ID: <
pascallanger/DIY-Multiprotocol-TX-Module/repo-discussions/823/comments/
***@***.***>
"
